packages/extension/src/ui/screens/HideTokenScreen.tsx

```diff
@@ -35,43 +26,33 @@ export const HideTokenScreen: FC = () => {
 
   const { name } = toTokenView(token)
 
-  return (
-    <>
-      <Header>
-        <BackButton />
-      </Header>
+  const handleSubmit = async () => {
+    try {
+      removeToken(token)
+      navigate(routes.account())
+    } catch (e) {
```

Comment:
Hide error variable if unused
```suggestion
    } catch {
```